docs/docs/development/local-toolchain/build-flash.mdx

```diff
@@ -95,40 +159,26 @@ west build -b nice_nano_v2 -- -DSHIELD=vendor_shield -DZMK_EXTRA_MODULES="C:/Use
 
 ### Building from `zmk-config` Folder
 
-Instead of building .uf2 files using the default keymap and config files, you can build directly from your [`zmk-config` folder](../../user-setup.mdx#github-repo) by adding
-`-DZMK_CONFIG="C:/the/absolute/path/config"` to your `west build` command. **Notice that this path should point to the folder labelled `config` within your `zmk-config` folder.**
+Instead of building .uf2 files using the default keymap and config files, you
+can build directly from your [`zmk-config` folder](../../user-setup.mdx#github-repo)
```

Comment:
```suggestion
can build using files from your [`zmk-config` folder](../../user-setup.mdx#github-repo)
```
This seems a bit wrong to me, since you aren't building while inside the config folder.

(I am aware this isn't coming from the new commits, but if we are touching here we can improve.)